docs/developers/signing-transactions/tools-for-signing.md

```diff
@@ -47,7 +47,7 @@ In either case, the output file looks like this:
 }
 ```
 
-## **Sign using [erdwalletjs-cli](/sdk-and-tools/erdwalletjs-cli)**
+## **Sign using [sdk-js-wallet-cli](/sdk-and-tools/sdk-js-wallet-cli)**
```

Comment:
The command alias below will change (can be left for future PR).

Reply:
changed